components/lib/utils/DomHandler.js

```diff
@@ -1,3 +1,5 @@
+import DOMPurify from 'isomorphic-dompurify';
```

Comment:
I used isomorphic-dompurify because it works serverside or client side so it works for Next SSR as well as client side.